.github/workflows/build-images.yml

```diff
@@ -243,103 +224,31 @@ jobs:
-
-    - name: Login to GitHub Container registry for non-PRs
-      uses: docker/login-action@v2
-      if: ${{ github.event_name != 'pull_request' && inputs.v2_vertica_image == '' || startsWith(inputs.v2_vertica_image, 'ghcr.io') }}
-      with:
-        registry: ghcr.io
-        username: ${{ github.repository_owner }}
-        password: ${{ secrets.GITHUB_TOKEN }}
-
-    - name: Login to Docker Hub
-      uses: docker/login-action@v2
-      if: ${{ inputs.v2_vertica_image != '' && startsWith(inputs.v2_vertica_image, 'docker.io') }}
-      with:
-        username: ${{ secrets.DOCKERHUB_USERNAME }}
-        password: ${{ secrets.DOCKERHUB_TOKEN }}
-
-    - uses: actions/checkout@v3
-      if: ${{ inputs.v2_vertica_image == '' }}
-
-    - name: Download the RPM
-      uses: ./.github/actions/download-rpm
-      if: ${{ inputs.v2_vertica_image == '' }}
-
-    - name: Build the v2 image
-      if: ${{ inputs.v2_vertica_image == '' }}
-      run: |
-        # We never push the image we build here because the RPM we use is too
-        # old. The RPM needs to be from at least a 23.3.0 server. So, we build
-        # here only to test the creation of the image. We don't verify this
-        # image in the remaining CI. Pick a dummy image name so as not to
-        # overwrite the output build image that the e2e tests will use.
-        export VERTICA_IMG=vertica-k8s:kind-v2
-        make docker-build-vertica-v2
-
-    - name: Do a local pull of the image if we didn't create it
-      if: ${{ inputs.v2_vertica_image != '' }}
-      run: |
-        docker pull ${{ inputs.v2_vertica_image }}
-
-    - name: Print a summary of the job
-      run: |
-        echo "Image Name: **${{ steps.v2_vertica_image.outputs.value }}**" >> $GITHUB_STEP_SUMMARY
-        echo "Was Built: ${{ inputs.v2_vertica_image == '' && '**Yes**' || '**No**' }}" >> $GITHUB_STEP_SUMMARY
-        echo "Was Pushed: ${{ inputs.v2_vertica_image == '' && github.event_name != 'pull_request' && '**Yes**' || '**No**' }}"
-        echo "Was Scanned: **No**" >> $GITHUB_STEP_SUMMARY
-        echo "Size: **$(docker inspect --format '{{.Size}}' ${{ steps.v2_vertica_image.outputs.value }} | numfmt --to=iec)**" >> $GITHUB_STEP_SUMMARY
-        echo "Image ID: **$(docker inspect --format '{{.ID}}' ${{ steps.v2_vertica_image.outputs.value }})**" >> $GITHUB_STEP_SUMMARY
-        echo "Digest: **$(IFS=":" read image tag <<< $(echo ${{ steps.v2_vertica_image.outputs.value }} | sed -e 's/^docker.io\///'); docker inspect --format='{{.RepoDigests}}' $image:$tag | sed 's:^.\(.*\).$:\1:' | tr " " "\n" | grep $image | cut -d'@' -f2 || echo "<none>")**" >> $GITHUB_STEP_SUMMARY
-        echo -n "Vertica Version: **" >> $GITHUB_STEP_SUMMARY
-        echo -n $(docker inspect --format '{{index .Config.Labels "vertica-version"}}' ${{ steps.v2_vertica_image.outputs.value }}) >> $GITHUB_STEP_SUMMARY
-        echo "**" >> $GITHUB_STEP_SUMMARY
-
   build-server-minimal:
     runs-on: ubuntu-latest
     outputs:
```

Comment:
I can't make a comment on the exact line, but a few lines down when we set `OWNER_LC`. That part can be removed.